graphql/schema.graphql

```diff
@@ -3,17 +3,27 @@ type Query {
   """
   Get customSessionKeys from Session
   """
-  customSessionKeys: String @cacheControl(scope: PRIVATE, maxAge: SHORT)
+  getOrderConfiguration: OrderConfig @cacheControl(scope: PRIVATE, maxAge: SHORT)
```

Comment:
I believe that is better to call this just "orderConfiguration", queries in graphql is always a "get" action